GliaWidgets/View/Chat/Entry/Upload/FileUploadStyle.swift

```diff
@@ -1,16 +1,46 @@
 import UIKit
 
+/// Style of a single upload view in the uploads list view.
 public class FileUploadStyle {
+    /// Style of the file image.
     public var fileImage: FileImageStyle
+
+    /// Style of the uploading state.
     public var uploading: FileUploadStateStyle
+
+    /// Style of the uploaded state.
     public var uploaded: FileUploadStateStyle
+
+    /// Style of the error state.
     public var error: FileUploadErrorStateStyle
+
+    /// Foreground color of the upload progress bar.
     public var progressColor: UIColor
+
+    /// Foreground color of the upload progress bar in error state.
     public var errorProgressColor: UIColor
+
+    /// Background color of the upload progress bar.
     public var progressBackgroundColor: UIColor
+
+    /// Image of the remove button.
```

Comment:
Similarly to other places where we give the opportunity to define an image - what kind of file it can be and what requirements apply to the image?

Reply:
same as https://github.com/salemove/ios-sdk-widgets/pull/27#discussion_r654463500